lib/generators/shopify_app/add_webhook/add_webhook_generator.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails/generators/base'
+
+module ShopifyApp
+  module Generators
+    class AddWebhookGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      class_option :topic, type: :string, aliases: "-t", required: true
+      class_option :address, type: :string, aliases: "-a", required: true
+
+      def init_webhook_config
+        initializer = load_initializer
+        return if initializer.include?("config.webhooks = [")
```

Comment:
That's a good idea. Ideally `load_initializer` (or a sister method) could load the config as a hash, and check if there is anything present for `config.webhooks`. Your call :)